static/scss/includes/variables.scss

```diff
@@ -6,3 +16,162 @@ $color-mpp-background: #E5E5E5;
 $color-mpp-gray-30: #ECECEC;
 
 $box-shadow-mpp-sm: 0px 0px 4px rgba(0, 0, 0, 0.12);
+
+
+// NEBULA COLORS
+
+// Primary Colors
+// Purple:
+$color-purple-90: #321C64;
+$color-purple-80: #45278D;
+$color-purple-70: #592ACB;
+$color-purple-60: #7542E5;
+$color-purple-50: #9059FF;
+$color-purple-40: #AB71FF;
+$color-purple-30: #C689FF;
+$color-purple-20: #CB9EFF;
+$color-purple-10: #D9BFFF;
+$color-purple-05: #E7DFFF;
+
+// Green:
+$color-green-90: #00736C;
+$color-green-80: #00A49A;
+$color-green-70: #1CC4A0;
+$color-green-60: #3AD4B3;
+$color-green-50: #3FE1B0;
+$color-green-40: #54FFBD;
+$color-green-30: #88FFD1;
+$color-green-20: #B3FFE3;
+$color-green-10: #D1FFEE;
+$color-green-05: #E3FFF3;
+
+// Neutral Colors
+// Grays:
+$color-black: #000000;
+$color-grey-60: #0C0C0D;
+$color-grey-50: #3D3D3D;
+$color-grey-40: #6D6D6E;
+$color-grey-30: #9E9E9E;
+$color-grey-20: #CECECF;
+$color-grey-10: #E7E7E7;
+$color-grey-05: #F9F9FA;
+
+// Whites:
+$color-white: #FFFFFF;
+$color-white-80: rgba($color-white, 0.8);
+$color-white-60: rgba($color-white, 0.6);
+$color-white-40: rgba($color-white, 0.4);
+$color-white-20: rgba($color-white, 0.2);
+
+
+// Functional Colors
+// Informational:
+$color-informational: #0060DF;
+$color-informational-active: #054096;
+$color-informational-hover: #0250BB;
+$color-informational-focus: #99BFF2;
+
+// Success:
+$color-success: #3AD4B3;
+$color-success-active: #1CC4A0;
+$color-success-hover: #B3FFE3;
+$color-success-focus: #3FE1B0;
+
+// Error
+$color-error: #FF4F5E;
+$color-error-active: #C50042;
+$color-error-hover: #E22850;
+$color-error-focus: #FFBDC5 ;
+
+// Warning
+$color-warning: #FFA436;
+$color-warning-active: #FFA436;
+$color-warning-hover: #FFA436;
```

Comment:
Should the :hover have a different color?

Reply:
Great catch. There should be four separate warning colors. Fixing now!